.github/workflows/ci.yml

```
@@ -266,6 +266,84 @@ jobs:
+          fi
+
+      - name: 📤 Upload Selenium test output
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: selenium-output-${{ github.run_id }}
+          path: selenium_output.txt
+          if-no-files-found: ignore
+          retention-days: 7
 
 # ───────────────────────────────────────────────────────────────────
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)